pingrid/impl.py

```diff
@@ -51,11 +51,11 @@
 from datetime import datetime, timedelta
 
 
-RAINBOW_COLORMAP = "[0x0000ff [0x00ffff 51] [0x00ff00 51] [0xffff00 51] [0xff0000 51] [0xff00ff 51]]"
+RAINBOW_COLORMAP = "[null 1973790 16711680 [16776960 51] [65280 51] [65535 51] [255 51] [16711935 51] 12632256]"
```

Comment:
Besides switching the byte order, you also added an extra entry at the start and another at the end. I'm guessing these are the colors to be used for values below the minimum and above the maximum, respectively. But parse_colormap doesn't seem to have a special case for those, so I'm not sure they'll be interpreted as intended. I'm thinking Igor might have removed them on purpose. Have you looked into this?

Reply:
I haven't... you're probably right. And then there are 3 values added to be removed: 2 at the start and 1 at the end, who respectively are the colors for missing, below min and above max.